packages/es-dev-server/src/create-middlewares.ts

```diff
@@ -38,10 +41,7 @@ function hasHook(plugins: Plugin[], hook: string) {
  * Creates middlewares based on the given configuration. The middlewares can be
  * used by a koa server using `app.use()`:
  */
-export function createMiddlewares(
-  config: ParsedConfig,
-  fileWatcher = chokidar.watch([]),
-): Middleware[] {
+export function createMiddlewares(config: ParsedConfig, fileWatcher = chokidar.watch([])) {
```

Comment:
return type is missing here, is that intentional?

Reply:
Intentional, it's inferred from the return statement.